packages/react-tooltip/Spec.md

```diff
@@ -1,63 +1,474 @@
+    </div>
+  </div>
+</div>
+```
 
-_Provide some representative example code that uses the proposed API for the component_
+# Migration
 
-## Variants
+## Migration from v8
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+The converged API does not support many of the custom features of the v8 tooltip. We may need to revisit and add additional features to the converged Tooltip if needed.
 
-## API
+- `Tooltip`
+  - `calloutProps` => Not supported. The arrow's visibility can be controlled using `noArrow`.
+  - `componentRef` => Not supported. The tooltip can't be manually invoked.
+  - `content="..."` => Either a component's `tooltip="..."` prop or the Tooltip's `children` prop.
+  - `delay` => `showDelay`
+  - `directionalHint` => `placement`
+    - `DirectionalHint.topLeftEdge` => `placement="top-start"`
+    - `DirectionalHint.topCenter` => `placement="top"`
+    - `DirectionalHint.topRightEdge` => `placement="top-end"`
+    - `DirectionalHint.topAutoEdge` => Not supported
+    - `DirectionalHint.bottomLeftEdge` => `placement="bottom-start"`
+    - `DirectionalHint.bottomCenter` => `placement="bottom"`
+    - `DirectionalHint.bottomRightEdge` => `placement="bottom-end"`
+    - `DirectionalHint.bottomAutoEdge` => Not supported
+    - `DirectionalHint.leftTopEdge` => `placement="left-start"`
+    - `DirectionalHint.leftCenter` => `placement="left"`
+    - `DirectionalHint.leftBottomEdge` => `placement="left-end"`
+    - `DirectionalHint.rightTopEdge` => `placement="right-start"`
+    - `DirectionalHint.rightCenter` => `placement="right"`
+    - `DirectionalHint.rightBottomEdge` => `placement="right-end"`
+  - `directionalHintForRTL` => Automatic based on whether the target element is RTL
+  - `maxWidth` => `style={{ maxWidth: ... }}`
+  - `onRenderContent` => Set `children` to a custom render function
+- `TooltipHost`
+  - `calloutProps` => Not supported
+  - `closeDelay` => `hideDelay` on the Tooltip
+  - `hostClassName` => N/A, not needed because there is no TooltipHost
+  - `onTooltipToggle` => Not supported
+  - `overflowMode` => `onlyIfTruncated`
+    - `TooltipOverflowMode.self` => `onlyIfTruncated="true"`
+    - `TooltipOverflowMode.parent` => Set `targetElement` to the parent element, and `onlyIfTruncated` to true
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+## Migration from v0
 
-## Structure
+- `Tooltip`
+  - `content="..."` => Either a component's `tooltip="..."` prop or the Tooltip's `children` prop.
+  - `defaultOpen` => Not supported
+  - `flipBoundary` => Not supported
+  - `mountNode` => ???
+  - `mouseLeaveDelay` => `hideDelay`
+  - `offset` => Not supported
+  - `onOpenChange` => Not supported
+  - `open` => Not supported
+  - `overflowBoundary` => Not supported
+  - `pointing` => `!noArrow`
+  - `popperRef` => Not supported
+  - `position` => `placement`
```

Comment:
```suggestion
  - `position` => `placement`
  - `align` => `placement`
```

v0 uses two props for position and alignment.... it can be hard to distinguish which controls what to be honest so having the single string could make sense, but regardless we should specify the replacement here